app/service/ingest.py

```diff
@@ -58,15 +60,17 @@ def save_collections(
 
 @validate_call(config=ConfigDict(arbitrary_types_allowed=True))
 def save_families(
-    family_data: list[dict], corpus_import_id: str, db: Optional[Session] = None
+    family_data: list[dict[str, Any]],
+    corpus_import_id: str,
+    db: Optional[Session] = None,
 ) -> list[str]:
     """
     Creates new families with the values passed.
 
-    :param list[dict] families_data: The data to use for creating families.
+    :param list[dict[str, Any]] families_data: The data to use for creating families.
```

Comment:
Incorrect parameter name.
```suggestion
    :param list[dict[str, Any]] family_data: The data to use for creating families.
```